City-of-Helsinki/drupal-hdbt
# UHF-1725: Supplementary button design change

To test:

1. Switch to this branch `composer require drupal/hdbt:dev-UHF-1725-supplementary-button-design-change`
2. Run `make drush-cr`
3. Go to add buttons to hero element with background image, banner paragraph, liftup with image paragraph, text paragraph, column paragraph and footer and check that they work consistently with all the designs (primary, secodary, supplementary) and that supplementary always has at least arrow icon on it (other icon choices and external links override the arrow).
